packages/rainbowkit/src/wallets/groupedWallets.ts

```diff
@@ -0,0 +1,62 @@
+import { WalletInstance } from './Wallet';
+
+export const groupedWallets = (
+  recentWallets: WalletInstance[],
+  walletInstances: WalletInstance[],
+) => {
+  return [
+    ...recentWallets,
+    ...walletInstances.filter(
+      (walletInstance) =>
+        !recentWallets.some((wallet) => wallet.id === walletInstance.id),
+    ),
+  ];
+};
+
+export const isRecentWallet = (
+  recentWallets: WalletInstance[],
+  walletId: string,
+) => {
+  return recentWallets.some((recentWallet) => recentWallet.id === walletId);
+};
+
+export const isRainbowKitConnector = (wallet: WalletInstance) => {
+  return !!wallet.isRainbowKitConnector;
+};
+
+export const isEIP6963Connector = (wallet: WalletInstance) => {
+  return !!(
+    !wallet.isRainbowKitConnector &&
+    wallet.icon &&
+    wallet.uid &&
+    wallet.name
+  );
+};
```

Comment:
This file contains all the helper function that we need in order to make rainbowkit connector + eip6963 work fine ✨